front/lib/api/assistant/generation.ts

```diff
@@ -83,6 +84,33 @@ export async function constructPromptMultiActions(
 
   // INSTRUCTIONS section
   let instructions = "INSTRUCTIONS:\n";
+
+  if (
+    model.providerId === "anthropic" &&
+    model.modelId === CLAUDE_3_5_SONNET_20241022_MODEL_ID
+  ) {
+    instructions += `\
+    STOP AND CHECK BEFORE USING VISUALIZATION:\n
+    MANDATORY CHECKLIST - ALL MUST BE TRUE:\n
+    □ Contains actual data (numbers, measurements, statistics)\n
```

Comment:
We definitely want to use markdown notation here otherwise you're biasing the model toward giving up on it.